UIKit/_UICache.h

```diff
@@ -0,0 +1,17 @@
+#import <Foundation/NSObject.h>
+
+NS_CLASS_AVAILABLE_IOS(7_0) @interface _UICache : NSObject
+
+- (instancetype)init;
```

Comment:
Is this line necessary?

Reply:
I suppose it is not, thanks, just edited it accordingly